lib/lexer.ml

```diff
@@ -127,7 +127,7 @@ let at_rule = [%sedlex.regexp? "@", ident]
 
 let at_rule_without_body = [%sedlex.regexp? "@", ("charset" | "import" | "namespace")]
 
-let nested_at_rule = [%sedlex.regexp? "@", ("document" | "keyframes" | "media" | "supports" | "scope")]
+let nested_at_rule = [%sedlex.regexp? "@", ("document" | "keyframes" | "media" | "supports" | "scope" | "-webkit-keyframes" )]
```

Comment:
I was wondering, should we add all the possible rules here, or only the most common ones?

Reply:
That's a great point! I'll cc @astrada so he sees this comment.

Reply:
Thanks! Feel free to add all the rules you need. When you are done, I will publish a new version on opam.

Reply:
@gaku-sei How many possible rules are out there? Adding the most common ones (at least to start) might be our best bet. If you can point me to a list of the ones you think I should add that'd be great!

Reply:
Sure, I could find a list on mdn: https://developer.mozilla.org/en-US/docs/Web/CSS/At-rule but it doesn't contain the vendor prefixes 😕 

I tried them all on https://autoprefixer.github.io/:

```css
/* input */
@charset {}
@import {}
@namespace {}
@media {}
@supports {}
@document {}
@page {}
@font-face {}
@keyframes {}
@viewport {}
@counter-style {}
@font-feature-values {}

/* output */
@charset {}
@import {}
@namespace {}
@media {}
@supports {}
@document {}
@page {}
@font-face {}
@-webkit-keyframes {}
@keyframes {}
@-ms-viewport {}
@-o-viewport {}
@viewport {}
@counter-style {}
@font-feature-values {}
```

Notice that some are still experimental, like document, and viewport.